packages/docs/src/routes/docs/(qwikcity)/endpoints/index.mdx

```diff
@@ -331,18 +331,22 @@ export const onGet = async (requestEvent: RequestEvent) => {
 Endpoints can manually write to the HTTP response stream, using the `getWritableStream()` method. This can be useful for creating a streaming endpoint, such as a server-sent events endpoint.
 
 ```tsx
-import type { RequestEvent } from '@builder.io/qwik-city';
+import type { RequestHandler } from '@builder.io/qwik-city';
 
-export const onGet: RequestHandler = async (requestEvent: RequestEvent) => {
+export const onGet: RequestHandler = async (requestEvent) => {
   const writableStream = requestEvent.getWritableStream();
   const writer = writableStream.getWriter();
-  writer.write('Hello World');
+  const encoder = new TextEncoder();
+  
+  writer.write(encoder.encode('Hello World'));
   await wait(100);
-  writer.write('After 100ms');
+  writer.write(encoder.encode('After 100ms'));
   await wait(100);
-  writer.write('After 200ms');
+  writer.write(encoder.encode('After 200ms'));
   await wait(100);
-  writer.write('END');
+  writer.write(encoder.encode('END'));
```

Comment:
While `TextEncoderStream` can also be used in this scenario, it appears that this section intends to demonstrate the `getWritableStream()` API. To achieve this, I have included the encoder in the example.
```ts
export const onGet: RequestHandler = async () => {
  const { writable } = new TextEncoderStream();
  const writer = writable.getWriter();
  writer.write('Hello World');
  await wait(100);
  writer.write('After 100ms');
  await wait(100);
  writer.write('After 200ms');
  await wait(100);
  writer.write('END');
  writer.close();
};

```